src/global/constants.js

```diff
@@ -9,6 +9,7 @@ export const BEM_PERSON_CORE_MILESTONES = 'person_core_milestones';
 export const BEM_PERSON_PANEL = 'person_panel';
 export const BEM_PERSON_PANEL_DETAILS = `${BEM_PERSON_PANEL}--details`;
 export const BEM_PERSON_PANEL_DETAILS_ACTION_BUTTON = `${BEM_PERSON_PANEL_DETAILS}_action_button`;
+export const BEM_POPOVER = 'popover';
```

Comment:
done

Reply:
Not done yet .... do you need to commit/push?

@morethanfire I'm curious as to why on in this particular case?  What about those lines make you want the space there?  Is it because Lines 9-11 (at least in the snippet shown here) are about Person Panel, and Line 12 is about a different thing (the Popover)?

Reply:
I marked it as I was fixing the code so I don't have to come back after pushing/commiting

Reply:
This appears fixed in 53d7c67.
Leaving the thread unresolved for @morethanfire 's benefit.